platform/src/main/java/org/stellar/anchor/platform/observer/ObservedPayment.java

```diff
@@ -88,6 +92,8 @@ public static ObservedPayment fromPathPaymentOperationResponse(
           (AssetTypeCreditAlphaNum) pathPaymentOp.getSourceAsset();
       sourceAssetCode = sourceIssuedAsset.getCode();
       sourceAssetIssuer = sourceIssuedAsset.getIssuer();
+    } else if (pathPaymentOp.getSourceAsset() instanceof AssetTypeNative) {
+      sourceAssetCode = "native";
```

Comment:
`"native"` is used many times. It maybe helpful to convert to a CONSTANT.

Reply:
Good point. Fixed!